venom-tests/api-json-validation.yml

```diff
@@ -0,0 +1,38 @@
+name:  Test Json Validation Plugin
```

Comment:
I believe this test suite is duplicated. There is also `venom-tests/QA-960-api-json-validation.yml` that runs the same test.